mod.test.ts

```diff
@@ -1245,8 +1245,11 @@ Deno.test("copy test", async () => {
       "cp: target 'non-existent' is not a directory\n",
     );
 
-    assertEquals(await getStdErr($`cp "" ""`), "cp: missing file operand\n");
-    assertStringIncludes(await getStdErr($`cp ${file1} ""`), "cp: missing destination file operand after");
```

Comment:
`cp file1 ""` should try to create a file `""` and fail, but it is considered a copy to `cwd`.
This is another issue.

```sh
$ cp file1 ""
cp: cannot create regular file '': No such file or directory
```